src/TelemetryEvents/DevHomeShellLoadedEvent.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Diagnostics.Tracing;
+using DevHome.Common.Helpers;
+using DevHome.Telemetry;
+using Microsoft.Diagnostics.Telemetry;
+using Microsoft.Diagnostics.Telemetry.Internal;
+using Microsoft.Windows.AppLifecycle;
+using Serilog;
+
+namespace DevHome.TelemetryEvents;
+
+[EventData]
+public class DevHomeShellLoadedEvent : EventBase
+{
+    public Guid DeploymentIdentifier { get; }
+
+    public ExtendedActivationKind ActivationKind { get; }

```

Comment:
I'm not sure the telemetry engine understands classes except for Exceptions.   I think.  How does this information show up in the telemetry?

Reply:
Good question. @vineeththomasalex or @jaholme is this something you know?